index.html

```diff
@@ -403,7 +402,7 @@ <h3>Requirements</h3>
 							<span>RECOMMENDED:</span>
 							<ul class="flat">
 								<li><a href="#wp-a11y">accessibility</a></li>
-								<li><a href="#wp-dir">base direction</a></li>
+								<li><a href="#wp-language">base direction</a></li>
```

Comment:
Why is this changed to language, it should still be dir?

Reply:
@lrosenthol the previous version contained two independent sections, one for language and another for direction. With the extra features it looked editorially better to merge the two sections. Hence the two identical targets. 

The "id" value may not be ideal (just kept the old id of one of the original sections) but that is only an internal thing, invisible to the reader. We can harmonize those later.